getting-started/proc-add-source-control-rhdh-catalog.adoc

```diff
@@ -51,34 +80,97 @@ data:
 --
 
 . Click *Save*.
-. Create a key or value Secret file named `rhdh-secrets` using the environment variables as keys from the previous code snippet by following the steps below:
+. Navigate to the *Helm* tab and select *Upgrade*.
+. Under *Root Schema → Backstage Chart Schema → Backstage Parameters → Backstage container environment variables from existing Secrets*, input `github-secrets` as the value.
+. Click *Upgrade*.
+
+== Configuring GitHub integration
+
+The configuration of GitHub is required to enable the GitHub plugins in {product-short}.
+
+.Procedure
+
+. In the Red Hat OpenShift cluster, navigate to the main page of the GitHub organization where you want to create the OAuth application.
+. Click *Settings* -> *<> Developer Settings* -> *GitHub Apps* -> *New GitHub App*.
+. Enter the application name as `Developer Hub`.
+. Add the following as the *Homepage URL*:
 +
 --
-.. In Red Hat OpenShift, go to the *Secrets* tab and click *Create*.
-.. Enter *Secret name* as `rhdh-secrets`.
-.. Add environment variables as *Key* and *Value* and click *Create*.
+`https://developer-hub-<NAMESPACE_NAME>.<OPENSHIFT_ROUTE_HOST>/`
 --
 
-. To log in using your GitHub application, ensure that the *Callback URL* in your GitHub application is configured as follows:
+. Add the following URL as *Authorization callback URL*:
 +
 --
 `https://developer-hub-<NAMESPACE_NAME>.<OPENSHIFT_ROUTE_HOST>/api/auth/github/handler/frame`
+--
+
+. Unselect *Webhook URL* -> *Active*.
+. Under the *Where can this GitHub App be installed?* section, ensure that *Only on this account* is selected.
```

Comment:
Does this step create the application? If so, it might improve scannability to add the result to line 109 and start line 110 with _Click_. Otherwise, it seems like there might be a step missing between lines 109 and 110.

Reply:
Done, added a step for clicking the register application button.